docs/submitting-clinical-data.md

```diff
@@ -2,15 +2,28 @@
 id: submitting-clinical-data
 title: Submitting Clinical Data
 ---
+Clinical data consists of all supporting data about your programs donors and their health data.
 
-## Uploading Data
+> Note: Before you can submit clinical data, your data must be assigned ARGO Identifiers through the sample registration process. See [Registering Samples](/s) for instructions on how to register samples.
 
-### How To
 
-### Links
+## How To Submit Clinical Data
+1. Navigate to the **Submission** area in the top menu.
+1. Click on the **Clinical Submission** section in the left menu pane for your program.
```

Comment:
Click on the **Submit Clinical Data** section in the left sidebar menu for your program.